server/main.lua

```diff
@@ -61,3 +62,73 @@ RegisterNetEvent('ox_police:setPlayerEscort', function(target, state)
 
     target:set('isEscorted', state and source, true)
 end)
+
+lib.callback.register('ox_police:spawnVehicle', function(source, model, mock)
+
+    -- perform checks to prevent exploits and abuse
+    local player = players[source]
+    local isClose = false
+    local isPolice = player.hasGroup(Config.PoliceGroups)
+
+    if not isPolice then
+        return print('player is not a policeman')
+    end
```

Comment:
The players table contains all players who are "in service", so there's no need to check if they have the necessary groups.
player will be nil if they aren't in service.